### ISSUE TITLE
[Lex] - `.nolist` in macros is not supported and causes a lexical error

### ISSUE THREAD

**Issue by harm-smits** [dude719/JetBrains-NASM-Language#27]
The following macro producs a lexical error:

```asm
%macro  multipush 1-*.nolist
	%rep  %0
		push    %1
		%rotate 1
	%endrep
%endmacro
```

The lex error is as follows:

```
<data>, <directive>, <instruction>, <label>, <preprocessor>, <structure>, NASMTokenType.COMMENT, NASMTokenType.CRLF, NASMTokenType.ELIF_TAG, NASMTokenType.ELSE_TAG, NASMTokenType.ENDIF_TAG, NASMTokenType.ID, NASMTokenType.MACRO_END_TAG, NASMTokenType.MACRO_PARAM_LBL_DEF, NASMTokenType.REGISTER, NASMTokenType.SECTION, NASMTokenType.SEGMENT, NASMTokenType.STRING or NASMTokenType.[ expected, got '.nolist'
```

**Comment by ajkhoury** [comment]
I'm already working on a fix for this. Will be out today hopefully! :)

I have been using your project to fix my broken bnf/lex

**Comment by ajkhoury** [comment]
Fixed in f79f0101dfbac8dd685fd0c3c79c22c95be9b65d
